app/src/main/java/com/boolder/boolder/view/detail/ProblemBSFragment.kt

```diff
@@ -90,6 +92,14 @@ class ProblemBSFragment(private val listener: BottomSheetListener) : BottomSheet
         updateLabels()
     }
 
+    // Hide bleau.info button if bleauInfoId if empty
+    private fun hideBleauButton(view: View) {
+        if(selectedProblem.bleauInfoId == "") {
```

Comment:
what if `bleauInfoId` is `null`?

Reply:
Good point, I used kotlin's native `isNullOrEmpty`  function instead